scripts/Packages/Package_NoLead__DFN_QFN_LGA_SON/size_definitions/qfn/qfn_texas.yaml

```diff
@@ -1059,7 +1059,7 @@ Texas_S-PX2QFN-N14:
     minimum: 1.95
     maximum: 2.05
   overall_height:
-    maximum: 0.36
+    maximum: 0.4
```

Comment:
Why not also include the minimum? It is given in the datasheet.

Reply:
done